core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1168,14 +1173,40 @@ class HttpGoogleServicesDAO(
   }
 
   private def getGroupServiceAccountCredential: Credential = {
-    new GoogleCredential.Builder()
-      .setTransport(httpTransport)
-      .setJsonFactory(jsonFactory)
-      .setServiceAccountId(clientEmail)
-      .setServiceAccountScopes(directoryScopes)
-      .setServiceAccountUser(subEmail)
-      .setServiceAccountPrivateKeyFromPemFile(new java.io.File(pemFile))
-      .build()
+    val credential = groupServiceAccountPool(Random.nextInt(groupServiceAccountPool.length))
+
+    //support both credential types in the short-term for the sake of backwards compatibility
```

Comment:
i think we should stop using pems completely (and i think devops wants us to). i don't want to completely remove pem support from rawls because it's a larger change so this allows our SA pool (which will exclusively use JSON creds) to co-exist with the current pem setup